packages/keyring-eth-ledger-bridge/src/ledger-keyring.ts

```diff
@@ -211,9 +210,11 @@ export class LedgerKeyring extends EventEmitter {
     this.hdPath = hdPath;
   }
 
-  async unlock(hdPath?: string, updateHdk = true): Promise<string> {
+  async unlock(hdPath?: string, updateHdk = true): Promise<Hex> {
     if (this.isUnlocked() && !hdPath) {
-      return 'already unlocked';
+      return this.#getChecksumHexAddress(
+        ethUtil.publicToAddress(this.hdk.publicKey, true).toString('hex'),
```

Comment:
Nit: What about using the same pattern for `ethUtil` than the Trezor's PR, where we import `publicToAddress`, etc.. ?

Reply:
Done in [6b04afc](https://github.com/MetaMask/accounts/pull/194/commits/6b04afc00591c9ef35d2403940963c27d69845e2)